runtest.lua

```diff
@@ -91,7 +90,10 @@ local short_opts = {
 	end,
 	["r:"] = function(opt_repr, arg)
 		reload_mode = true
-		test_name = arg
+		if explicit_tests == nil then
```

Comment:
This is silly, test_name was used before because it was convenient and aligned with the single test mode. If the single test mode isn't single anymore, this should use a new `reload_name` instead, because reload mode only makes sense with 1 file.